cmake/modules/FindTagLib.cmake

```diff
@@ -13,17 +13,51 @@
 #
 #   TagLib::TagLib   - The TagLib library
 
-if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_TAGLIB taglib>=1.9.0 QUIET)
-endif()
+if(ENABLE_INTERNAL_TAGLIB)
+  include(cmake/scripts/common/ModuleHelpers.cmake)
+
+  set(MODULE_LC taglib)
+
+  SETUP_BUILD_VARS()
+
+  set(TAGLIB_VERSION ${${MODULE}_VER})
+
+  if(WIN32 OR WINDOWS_STORE)
+    # find the path to the patch executable
+    find_program(PATCH_EXECUTABLE NAMES patch patch.exe REQUIRED)
```

Comment:
Should we use the same code as in `cmake/scripts/windows/tools/patch.cmake` ?

https://github.com/xbmc/xbmc/blob/342ee328cef1dc8f59c9fee92ac945f6afc6497f/cmake/scripts/windows/tools/patch.cmake#L2-L8

Reply:
The windows patch stuff needs to be factored out info a find module. Haven't gotten around to it as yet. Will handle at a later time